help/deploy/releases/release-history.md

```diff
@@ -20,184 +20,159 @@ position: 6 # float position is supported
-* (Improvement) GROK-11494: Widgets: events (WIP)
-* (Improvement) GROK-16278: Avoid sending extra data in files serialization 
-* (Improvement) GROK-16279: Make browse less chatty (WIP)
-* (Bug) GROK-16275: #2771: Viewers: filter out legend with empty categories in the legend 
-* (Bug) GROK-16277: #2934: (Mac OS) Grid: some hotkeys for selection/deselection are not working for columns 
-* (Improvement) GROK-16254: #2932: Viewers: hide the option “Is Grid” from property panel 
-* (Bug) GROK-16320: Predictive modeling: interactive model saved without a name (WIP)
-* (Bug) GROK-16309: Set tooltip dialog looks bad 
-* (Bug) GROK-16331: Core: Bar Chart: molecules are renderered incorrectly on resizing bar chart 
-* (Bug) GROK-16330: Core: Table view: Form styles with refresh button is broken 
+### Grid
+* Improvements:
+  * GROK-16205: Support markup links. 
+  * GROK-15984: Formula-based rendering.
+  * [#2789](https://github.com/datagrok-ai/public/issues/2789): Ability to display the table name/description on the Grid.
+* Fixes:
+  * GROK-15188: Dataframe/grid: renaming column from '~name' -> 'name' doesn’t change visibility of column. 
+  * [#2934](https://github.com/datagrok-ai/public/issues/2934): Some hotkeys for selection/deselection are not working for columns (MacOS).
+ 
+### [Scatterplot](../../visualize/viewers/scatter-plot.md)
+* Improvements and fixes:
+  * GROK-14632: Marker legend isn't rendered in the trellis plot. 
+  * GROK-14376: Scatterplot: unzooming with mouse scroll can go to infinity.
+  * [#2913](https://github.com/datagrok-ai/public/issues/2913): Improve different context menu style / functionality between Scatterplot in Trellis vs native Scatterplot.
+  
+### [Line Chart](../../visualize/viewers/line-chart.md)
+* Fixes:
+  * GROK-15466: Table > Add View: Formula Lines for Line Chart should be present in added view.
+  * GROK-15464: Receiving errors when setting the second split in some cases. 
+  * GROK-15485: Bugs in RowSource = FilteredSelected, SelectedOrCurrent.
+  * [#2875](https://github.com/datagrok-ai/public/issues/2875): Line chart, trellis plot: number of selected columns is not updated in the properties panel immediately. 
+  * [#2904](https://github.com/datagrok-ai/public/issues/2904): When multi axis option is used. together with split, line chart is empty. 
+  
+### Bar chart
+* Improvement:
+  * [#2881](https://github.com/datagrok-ai/public/issues/2881): Bar chart: add an option to rotate / invert axes. 
+* Fixes:
+  * GROK-15887: Viewer window can't be resized in height when it's too high on the screen. 
+  * GROK-16234: Wrong label shortening. 
+  * GROK-16331: Molecules are rendered incorrectly on resizing bar chart. 
+  
+### Box plot
+* Improvements:
+  * [#2905](https://github.com/datagrok-ai/public/issues/2905): Add ''Size by" or "Shape by" or "color by'' options. 
+  * GROK-9748: Violin-state option. 
+* Fix:
+  * GROK-16244: Violin style: category bug. 
```

Comment:
If we introduce a feature in this release, why list the fixed bug related to that feature?

Reply:
removed bug mentioning